script/get-all-versions.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Script combines collection of latest versions of operators and images used in values.yaml (values-tools.yaml) and
+# outputs newest frozen values
+
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+tmpdir=$(mktemp -d)
+trap "rm -rf $tmpdir" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)